styles.css

```diff
@@ -80,8 +70,7 @@ body {
     }
```

Comment:
Bij een @media hoeft je de dingen die hetzelfde blijven niet nogmaals er in te zetten. Bijvoorbeeld de grid-gap en kleuren blijft bij zowel desktop als bij mobiel hetzelfde.